contributor/01-introduction-article.asciidoc

```diff
@@ -12,60 +12,52 @@ If you're not getting what you need, you can make the change you need directly i
+You might work with the product owner role to make sure that general aspects of documentation, or consistency of UI/UX, are kept upon merging your contribution.
 
+Last but not least, someone acting as product owner might have been involved in bringing the project, its benefits and community to your attention.
 
-If you want to learn in more detail what these other roles are about, and we encourage you to do so, we've prepared separate sections about the [_Trusted Committer_][TC Introduction] and the [_Product Owner_][PO Introduction].
+If you want to learn in more detail what these other roles are about, and we encourage you to do so, we've prepared separate sections about the https://github.com/InnerSourceCommons/InnerSourceLearningPath/blob/master/trusted-committer/01-introduction.asciidoc[_Trusted Committer_] and the https://github.com/InnerSourceCommons/InnerSourceLearningPath/blob/master/product-owner/01-opening-article.asciidoc[_Product Owner_].
 
+=== Section overview
 
-## Section overview
-In the following 5 segments you will learn more in detail about the various aspects introduced here. 
+In the following 5 segments you will learn more in detail about the various aspects introduced here.
 
 The next segment will detail _mindset and habit_ that create opportunities to _become an InnerSource Contributor_.
 
 In the third segment we'll look into the _ethos of the Contributor_ - i.e. aspects of _behavior_ that will lead to a pleasant and productive time for you and the host team, and might spark more collaboration.
-The guest-in-home analogy presented in the introductory videos will serve as a vivid example. 
+The guest-in-home analogy presented in the introductory videos will serve as a vivid example.
 
 The fourth segment describes the hands-on things to do to make your contribution a success - the _mechanics of Contributing_.
-We'll give practical tips to leverage when preparing to work on a contribution, during development, and also in the pull request.  
+We'll give practical tips to leverage when preparing to work on a contribution, during development, and also in the pull request.
 
 After we've dealt with the personal, the interaction-focused, and the technical aspects of the contributor role, the fifth segment presents the benefits of making the effort to contribute.
 We'll show benefits from various perspectives: yours, your team's, and the perspective of the company at large.
 
-The last segment will recap what we've learned about being an InnerSource contributor. 
+The last segment will recap what we've learned about being an InnerSource contributor.
 We'll share how you can continue your learning of InnerSource both with other online videos and articles, and through involvement with the online InnerSource community.
-
-
-[intro-innersource]:https://www.safaribooksonline.com/videos/introduction-to-innersource/9781492041504
```

Comment:
These multi-use link definitions not allowed in ASCIIDOC?  Too bad.

Reply:
At least kramdoc does not convert them to multi-use ones. Maybe @nadamsoreilly knows more about this (or can point to a the relevant documentation).

Reply:
Are multi-use links the same as "Reference Style links": https://www.markdownguide.org/basic-syntax/#reference-style-links?

Unfortunately, the auto-conversion scripts do not seem to be able to parse these links (I tried it with Pandoc and got the same outcome).

Asciidoc has a similar feature which they call "attributes" https://asciidoctor.org/docs/asciidoc-writers-guide/#document-attributes. That might be useful for setting up similar shortcuts once the files are in Asciidoc.

Reply:
That's great.